featuros-plugin/build.gradle

```diff
@@ -1,2 +1,9 @@
 project(':featuros-plugin') {
+    dependencies {
+        compile 'org.jodd:jodd-http:3.6.6'
```

Comment:
As usual, _it depends_. If you're using groovy already, then there's probably less to be gained by jodd (could give it a shot though). For an open source library it's rather a good lightweight choice that doesn't bring a ton of dependencies with it. On functional side, it's limited (no async), but it fits very well for now.

[Original appearance on HN](https://news.ycombinator.com/item?id=9278704)